src/app/containers/TestViewWrapper/index.tsx

```diff
@@ -0,0 +1,87 @@
+import classNames from "classnames/bind";
+import { useParams } from "react-router-dom";
+import { Typography } from "@mui/material";
+
+import styles from "./TestViewWrapper.module.scss";
+import { BreadcrumbsCustomization } from "app/components/BreadcrumbsCustomization";
+import { useGetQuestionsByExamId } from "queries/question";
+import RenderQuestion from "app/components/RenderQuestion";
+
+const cx = classNames.bind(styles);
+
+export const TestViewWrapper = () => {
+  const { examId } = useParams();
+  const { data: questionsByExamId } = useGetQuestionsByExamId(examId as string);
+
+  return (
+    <div className={cx("container")}>
+      <BreadcrumbsCustomization
+        className={cx("breadcrumbs")}
+        current="Chi tiết"
+        breadcrumbsList={[{ linkTo: "/exams", text: "Các môn học" }]}
+      />
+
+      <div className={cx("examInfo")}>
+        <Typography className={cx("examName")} component="h1">
+          {questionsByExamId?.exam?.title}
+        </Typography>
+
+        <div className={cx("examItem")}>
+          <Typography className={cx("nameItem")} component="p">
+            Môn học
+          </Typography>
+          <Typography className={cx("colon1")} component="p">
+            :
+          </Typography>
+          <Typography className={cx("highlight")} component="strong">
+            {questionsByExamId?.exam?.subject?.subject_name}
+          </Typography>
+        </div>
+
+        <div className={cx("examItem")}>
+          <Typography className={cx("nameItem")} component="p">
+            Tác giả đăng
+          </Typography>
+          <Typography className={cx("colon2")} component="p">

```

Comment:
Colon2?

Reply:
there's a difference between the colons so I've given them different classnames :  colon1 and colon2. That shouldn't matter much, will it?

Reply:
oke